docs/source/swift-scripting.md

```diff
@@ -247,12 +243,12 @@ Now, you're able to generate code from a debuggable Swift Package Manager execut
     swift run
     ```
     
-    >**NOTE**: If your package ever seems to have problems with caching, run `swift package clean` before `swift run` for a totally clean build. It is not recommended to do this by default since it substantially increases build time.
+    >**Note**: If your package ever seems to have problems with caching, run `swift package clean` before `swift run` for a totally clean build. It is not recommended to do this by default, because it substantially increases build time.
     
-4. Build your target. Since `swift run` is being called from within your target, all of the pieces of the environment, including `$SRCROOT`, will automatically be passed to the environment of the executable, and you don't have to worry about passing anything manually. 
+4. Build your target. Because `swift run` is called from within your target, all of the pieces of the environment, including `$SRCROOT`, are automatically passed to the environment of the executable, and you don't have to worry about passing anything manually. 
```

Comment:
I think I can remove everything after `build your target` here - the SRCROOT thing is no longer necessary.